tests/unit/commands/test_remote.py

```diff
@@ -163,7 +164,7 @@ def test_no_confirmation_for_private_project(
 
 
 @pytest.mark.parametrize("base", const.CURRENT_BASES)
-@pytest.mark.usefixtures("mock_argv")
+@pytest.mark.usefixtures("mock_argv", "emitter")
```

Comment:
I'm confused about the combination of adding the emitter fixture above with adding it here - surely we should only need one or the other?

Reply:
pytest.mark.usefixtures overrides instead of inherit, it also does not work on fixture declaration :shrug: so I have learned

Reply:
Well that's fun! Thanks for the explanation